packages/product-backend/src/service.ts

```diff
@@ -45,6 +46,14 @@ export const createProductAccounting = async (p: {
   if (!process.env.PRODUCT_BACKEND_URL) {
     throw new Error('No product backend URL set')
   }
+  // These values are required by SAP and need be provided
+  if (
```

Comment:
These should be added to yup schema, no need to do custom validations

Reply:
These validations are done by controller, so it is easy to add to there.

Reply:
The more you know - this is cleaner and much simpler, thanks!